### PR TITLE
Pagopa 1011 creazione schedule per lettura table e aggiornamento file xml

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Adding scheduler to update the ica binary file table accordingly to the design review, creating the xml and inserting it in the corresponding table.
PR tied to https://github.com/pagopa/pagopa-api-config-starter/pull/23

#### Motivation and Context
This was needed due the iban refactoring. It will be dismissed with the dismissal of the ICA

#### How Has This Been Tested?
This has been tested manually using the h2 profile and with junit

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FedericoRuzzier** [comment]
update_code

File: src/main/java/it/gov/pagopa/apiconfig/core/scheduler/SchedulerIca.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    return outputStream.toByteArray();
+  }
+
+  private void writeElement(XMLStreamWriter xtw, String elementName, String elementValue) {
+    try {
+      xtw.writeStartElement(elementName);
+      xtw.writeCharacters(elementValue);
+      xtw.writeEndElement();
+    } catch (XMLStreamException e) {
+      log.error(XML_ERROR, e.getMessage());
```
Comment (FedericoRuzzier) [inline src/main/java/it/gov/pagopa/apiconfig/core/scheduler/SchedulerIca.java:121]: throw AppException

File: src/test/resources/application.properties [hunk 2]
```diff
@@ -48,5 +48,13 @@ service.utils.subscriptionKey=subscriptionKey
 # Nodo monitoring configuration
 service.nodo-monitoring.host=http://localhost:8587
 
+# Scheduling configuration
+cron.job.schedule.enabled=true
```
Comment (FedericoRuzzier) [inline src/test/resources/application.properties:52]: put to false

File: src/main/resources/application-h2.properties [hunk 3]
```diff
@@ -53,5 +53,13 @@ retry.utils.maxDelay=200
 # Nodo monitoring configuration
 service.nodo-monitoring.host=http://localhost:8587
 
+# Scheduling configuration
+cron.job.schedule.enabled=true
```
Comment (FedericoRuzzier) [inline src/main/resources/application-h2.properties:57]: Disable

**Review by jacopocarlini — APPROVED** [review]
Good Job!

**Pull request merged**